mappings/platform_file_centric.json

```diff
@@ -0,0 +1,317 @@
+{
+  "aliases": {
+    "file_centric": {}
+  },
+  "settings": {
+    "index.number_of_shards": 3,
+    "index.max_results_window": 300000,
+    "analysis": {
+      "analyzer": {
+        "autocomplete_analyzer": {
+          "filter": ["lowercase", "edge_ngram"],
+          "tokenizer": "standard"
+        },
+        "autocomplete_prefix": {
+          "filter": ["lowercase", "edge_ngram"],
+          "tokenizer": "keyword"
+        },
+        "lowercase_keyword": {
+          "filter": ["lowercase"],
+          "tokenizer": "keyword"
+        }
+      },
+      "filter": {
+        "edge_ngram": {
+          "max_gram": 20,
+          "min_gram": 1,
```

Comment:
Just a thought, not sure we need go as low as `1`, it increases index size and maybe memory consumption in ES server. Maybe it's negligible.

Reply:
I found that what was used in ICGC in the dcc release index, but we can tune this for sure I think 2 or 3 may work as well but I won't go higher than 3

Reply:
Personally `2` works well. In some biological terms, like gene symbol, they are typically short, like 2-5 characters. If `min_gram` sets to 3, some genes will not be searchable by autocompletion.

Reply:
so do we want to go with 2 for now @blabadi or stick with 1 till we need to adjust?

Reply:
yeah 2 works will update